app/page/index.js

```diff
@@ -69,6 +81,58 @@ Page(
           });
         },
       });
+
+      const text = createWidget(widget.TEXT, {
+        x: 240,
+        y: 90,
+        w: 150,
```

Comment:
can this constant object (styling) go in the constants file like how the buttons' style is, so this file's easier to parse